cluster/gce/config-test.sh

```diff
@@ -49,7 +49,7 @@ CLUSTER_NAME="${CLUSTER_NAME:-${INSTANCE_PREFIX}}"
 MASTER_NAME="${INSTANCE_PREFIX}-master"
 MASTER_TAG="${INSTANCE_PREFIX}-master"
 NODE_TAG="${INSTANCE_PREFIX}-minion"
-CLUSTER_IP_RANGE="${CLUSTER_IP_RANGE:-10.245.0.0/16}"
+CLUSTER_IP_RANGE="${CLUSTER_IP_RANGE:-10.245.0.0/14}"
```

Comment:
I'd prefer that we change it to `10.248.0.0/16` so that it maintains the current intent of having a separate range for test/non-test by default.